x-pack/filebeat/input/azureeventhub/v1_input_test.go

```diff
@@ -0,0 +1,7 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build !aix
+
+package azureeventhub
```

Comment:
Empty file? Did something get lost when copying/moving the code or is this file not need any more?
```suggestion
```

Reply:
I considered moving some of the tests here but then changed my mind. Thanks for pointing this out. I'm removing it!